content/en/docs/reference/config/analysis/ist0002/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Deprecated
+layout: analysis-message
+---
+
+This message occurs when you use a function that is planned for removal in a future version.
```

Comment:
"function" is an overloaded term. How about "feature" instead?

Reply:
Great, done!